src/sugar3/graphics/style.py

```diff
@@ -156,52 +156,78 @@ def zoom(units):
     return int(ZOOM_FACTOR * units)
 
 
-ZOOM_FACTOR = _compute_zoom_factor()
+ZOOM_FACTOR = _compute_zoom_factor()  #: scale factor, as float (eg. 0.72, 1.0)
 
-DEFAULT_SPACING = zoom(15)
-DEFAULT_PADDING = zoom(6)
-GRID_CELL_SIZE = zoom(75)
-LINE_WIDTH = zoom(2)
+DEFAULT_SPACING = zoom(15)  #: Spacing is placed in-between components
+DEFAULT_PADDING = zoom(6)  #: Padding is placed around a ui component
+GRID_CELL_SIZE = zoom(75)  #: Grid cells are the fundamental measure for sugar
```

Comment:
> are the fundamental measure

Of what?  The comment doesn't say what they are a measure of or why they are fundamental.